### PR TITLE
Use color squares for type dropdown in contract editor [finishes #89659886]

### PR DESCRIPTION
**Pull request by bcjordan**
# After            |          Before

image

### CODE REVIEW

**Comment by breville** [comment]
have we discussed variations of any of these colours?  i find them all a strong set but unusual in combination.

**Comment by bcjordan** [comment]
@breville don't think we have yet, let's bring it up in the room

File: blockly-core/core/ui/contract_editor/type_dropdown.js [hunk 1]
```diff
@@ -76,12 +85,22 @@ Blockly.TypeDropdown.prototype.createSelect_ = function() {
...
   newTypeDropdown.setValue(this.type_);
   return newTypeDropdown;
 };
 
+/**
+ * @returns {Element}
+ * @private
+ */
+Blockly.TypeDropdown.prototype.createColorSquareDiv = function () {
+  return goog.dom.createDom('div', "color-square-icon");
```
Comment (Bjvanminnen) [inline blockly-core/core/ui/contract_editor/type_dropdown.js]: Doesn't really matter, but feels strange switching between single and double quotes on the same line here.

**Comment by Bjvanminnen** [comment]
lgtm. the squares look good! :)

**Comment by bcjordan** [comment]
Tested in Safari, Firefox (good aside from now-known issue #92177128) and IE10, look fine across them all

**Pull request merged**